themes-default/slim/package.json

```diff
@@ -26,7 +26,9 @@
       }
     }
   },
-  "dependencies": {},
+  "dependencies": {
+    "babel-polyfill": "6.26.0"
```

Comment:
1. This is an old package name, new name is `@babel/polyfill`
2. All dependencies should be `devDependencies` (added using `yarn add -D <pkg>`)
3. `@babel/polyfill` is already a `devDependency` (line 37)